rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc

```diff
@@ -55,8 +64,17 @@ include::modules/rosa-hcp-vpc-manual.adoc[leveloffset=+2]
 * link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-getting-started.html[Get Started with Amazon VPC]
 * link:https://developer.hashicorp.com/terraform[HashiCorp Terraform documentation]
 
-include::modules/rosa-hcp-byo-oidc.adoc[leveloffset=+1]
-//include::modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc[leveloffset=+2]
+[id="rosa-hcp-byo-odic-overview_{context}"]
+== Creating an OpenID Connect Configuration
+
+When using a Red-Hat hosted cluster, you can create a managed or unmanaged OpenID Connect (OIDC) configuration that is generated by the CLI. With a managed OIDC configuration, it is stored within Red Hat's AWS account. A generated unmanaged OIDC configuration is stored in your AWS account and registers the intention to use that configuration with OCM. This generated OIDC configuration provides the private key for you to access. This process does not provide a private key for users to access.
```

Comment:
Should this be "Red Hat"? Or maybe you meant "Red Hat-hosted"? Also suggestion to include "unmanaged" here because its kind of confusing to parse which one is being referred to:
```suggestion
When using a Red Hat hosted cluster, you can create a managed or unmanaged OpenID Connect (OIDC) configuration that is generated by the CLI. With a managed OIDC configuration, it is stored within Red Hat's AWS account. A generated unmanaged OIDC configuration is stored in your AWS account and registers the intention to use that configuration with OCM. This generated unmanaged OIDC configuration provides the private key for you to access. This process does not provide a private key for users to access.
```
also, does the phrase "registers the intention to use" mean something specific? It is also a little confusing to understand what is meant here